packages/config/config/devices/0x0129/yrd420-f-zw3.json

```diff
@@ -0,0 +1,62 @@
+{
+	"manufacturer": "Yale",
+	"manufacturerId": "0x0129",
+	"label": "YRD420-F-ZW3",
+	"description": "Assure 2 Lock Touch SL Key Free Touchscreen Deadbolt",
+	"devices": 
```

Comment:
missing `[`
`"devices":` should be `"devices": [`
otherwise the closing bracket for the array on line 11 will break the code.